### DISCUSSION TITLE
update "places" on mulitple photos at once

### DISCUSSION THREAD

**Discussion by Nubicola** [photoprism/photoprism#3564] [Q&A]
Hi,

I'd like to set the "place" for numerous photos at the same time. It looks like I have to set the latitude and longitude. I'm not bothered if all the photos are at the same lat/long (I mean...it's good enough for me). However, it seems that despite selecting numerous photos in the UI, when I edit them, I have to change the lat/long one photo at a time.

How can I change the lat/long of a whole batch?

Thanks

**Comment by lastzero** [comment]
At the moment you need to use external software like https://www.digikam.org/ for this (and then re-index). We will also add a batch editing feature once we have released support for 2FA and OpenID Connect, so maybe later this year.
